picoquic/picoquic_internal.h

```diff
@@ -775,6 +775,14 @@ void picoquic_queue_stateless_reset(picoquic_cnx_t* cnx,
 
 picoquic_misc_frame_header_t* picoquic_create_misc_frame(const uint8_t* bytes, size_t length);
 
+int stream_reset_sent(picoquic_stream_head* stream);
+int stream_reset_requested(picoquic_stream_head* stream);
+int stream_stop_sending_requested(picoquic_stream_head* stream);
+int stream_stop_sending_sent(picoquic_stream_head* stream);
+int stream_stop_sending_received(picoquic_stream_head* stream);
+int stream_fin_notified(picoquic_stream_head* stream);
+int stream_fin_sent(picoquic_stream_head* stream);
+
```

Comment:
See previous remark on macros vs functions. You could easily replace these function declarations by macros.